pkgs/top-level/all-packages.nix

```diff
@@ -18595,6 +18595,8 @@ with pkgs;
 
   openocd = callPackage ../development/embedded/openocd { };
 
+  openocd-esp32 = callPackage ../by-name/es/esp-idf/openocd-esp32.nix { };
```

Comment:
If you want to expose these packages publicly, then they should go into their own folder like `by-name/op/openocd-esp32/packages.nix`. Then they will also be imported automatically.

Reply:
oh, I see this could make trouble because you are referencing the `source-info.json` from all these packages.

Reply:
exactly. But I'm open to alternatives, but I think it is more clear this way. I don't know if we have more precise guidelines on this topic.